src/components/stepper/stepper.e2e.ts

```diff
@@ -577,4 +577,30 @@ describe("calcite-stepper", () => {
       });
     });
   });
+
+  it("should render correct numbering-system with multiple stepper component", async () => {
+    const page = await newE2EPage();
+    await page.setContent(html`<calcite-stepper numbered>
+      <calcite-stepper-item heading="Add info" description="Subtitle lorem ipsum" complete id="step-one"
+        >Step 1 Content here lorem ipsum</calcite-stepper-item
+      >
+    </calcite-stepper>
+
+    <calcite-stepper numbered numbering-system="arab" lang="ar" dir="rtl" >
+      <calcite-stepper-item heading="الخطوةالاولى" complete>
+       الخطوة الأولى للمحتوى هنا
+    </calcite-stepper>`);
+    const [stepper1, stepper2] = await page.findAll("calcite-stepper");
+    expect(stepper2.getAttribute("numbering-system")).toEqual("arab");
+
+    await stepper1.click();
+    await page.waitForChanges();
+    await stepper2.click();
+    await page.waitForChanges();
+    await stepper1.click();
+    await page.waitForChanges();
```

Comment:
Main reason behind adding clicks is to make sure we reproduce the error state. Sometimes, on initial render the error may appear.